package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "play",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Perhaps 1.0.2 instead?

Reply:
This is totally something Trent would say!